src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import axios from "axios";
+import "./SingleArticle.css";
+
+const instance = axios.create({
```

Comment:
Implementing the feedback I left on articles.jsx will avoid having to repeat this line for example 🙂